test/integration/test_integration_rioxarray.py

```diff
@@ -484,6 +515,32 @@ def test_clip_geojson(open_func, from_disk):
             clipped_ds.rio.transform(),
             (3.0, 0.0, 425500.68381405267, 0.0, -3.0, 4615477.040546387, 0.0, 0.0, 1.0),
         )
+        # check for variables without spatial dims
```

Comment:
This looks like a duplicate of the above test. If that is the case, it can be removed. If not, mind making it a separate test?

Reply:
It is the same just with `rio.clip_box` and `rio.clip`.
How about combining both into `def test_clip_clip_box__non_geospatial()`?

Reply:
I would actually prefer two separate tests for the future as it narrows down the debugging.

To reduce duplication, you could create a function that generates the dataset that you could share with the two tests.

Reply:
However, what you have is fine if you would rather not change it.

Reply:
I've just separated the functions, so if thats okay with you, its fine with me! 
Thanks!